Cesium3DTilesSelection/src/LayerJsonTerrainLoader.cpp

```diff
@@ -554,28 +561,37 @@ LayerJsonTerrainLoader::createLoader(
                 pResponse->data(),
                 useWaterMask);
           })
-      .thenInMainThread([](LoadLayersResult&& loadLayersResult) {
-        return convertToTilesetContentLoaderResult(std::move(loadLayersResult));
+      .thenInMainThread([ellipsoid](LoadLayersResult&& loadLayersResult) {
+        return convertToTilesetContentLoaderResult(
+            ellipsoid,
+            std::move(loadLayersResult));
       });
 }
 
 CesiumAsync::Future<TilesetContentLoaderResult<LayerJsonTerrainLoader>>
 Cesium3DTilesSelection::LayerJsonTerrainLoader::createLoader(
+    const TilesetExternals& externals,
```

Comment:
I'd suggest just passing in the Ellipsoid itself, rather than the entire TilesetExternals. That way the TilesetContentManager doesn't need to lambda-capture the entire `TilesetExternals` instance.